static/js/containers/DashboardPage.js

```diff
@@ -438,6 +450,25 @@ class DashboardPage extends React.Component {
     dispatch(setRecentlyAttachedCoupon(coupon));
   };
 
+  navigateToProfile = () => {
```

Comment:
i moved this function and the one below to keep the `render*` functions together